bosh/releases/pre_render_scripts/api/routing-api/bpm/patch_bpm.sh

```diff
@@ -5,15 +5,18 @@ set -o errexit -o nounset
 target="/var/vcap/all-releases/jobs-src/routing/routing-api/templates/bpm.yml.erb"
 
 PATCH=$(cat <<'EOT'
-@@ -11,7 +11,7 @@
+@@ -11,8 +11,10 @@
      - -timeFormat
      - rfc3339
      - -ip
 -    - <%= spec.ip %>
-+    - 0.0.0.0
++    - "$(POD_IP)"
      <% if p("routing_api.auth_disabled") == true %>- -devMode <% end %>
++    env:
++      POD_IP: 0.0.0.0 # Set by k8s using an ops-file with cf-operator.
```

Comment:
Not sure I understand why we add an env to the bpm process.

Reply:
It's a bug on cf-operator right now. Should go away after it's fixed there.